rust/language-reference/7-tests.md

```diff
@@ -135,5 +136,37 @@ Since integration tests sit in a completely different crate than our project, th
 Also, it's important to notice that it's necessary for the project to have a `lib.rs` file (i.e. to be exposed as a library) for integration tests to work, because otherwise it would be impossible to import the project as an external crate inside the `tests` crate.
 
 
+## Documentation tests
+
+These tests can be a quick way to embed documentation *and* unit tests at the same time.
+These tests have a special `//!` comment placed at the module level and can be run with `cargo tests`.
```

Comment:
What happens when we run `cargo test` (I believe there's a typo too) with these tests in place?

Reply:
acknowledge the typo, sorry. These "documentation" tests will be run together along your tests marked by the usual `#cfg[test]` tag.